src/server/auth/server/oidc.go

```diff
@@ -84,31 +86,19 @@ func half(state string) string {
 // structs that we would normally need to instantiate multiple times
 type oidcConfig struct {
 	*auth.OIDCConfig
-	oidcProvider  *oidc.Provider
-	oauthConfig   oauth2.Config
-	rewriteClient *http.Client
+	oidcProvider *oidc.Provider
+	oauthConfig  oauth2.Config
 }
 
 func newOIDCConfig(ctx context.Context, config *auth.OIDCConfig) (*oidcConfig, error) {
-	var rewriteClient *http.Client
-	var err error
-	if config.LocalhostIssuer {
```

Comment:
Some version of this code is still going to be necessary because by default minikube doesn't support hairpin network routes, so a pod in the pachd service can't access itself (https://kubernetes.io/docs/tasks/debug-application-cluster/debug-service/#a-pod-fails-to-reach-itself-via-the-service-ip)